### PR TITLE
Automatic versioning with release drafter

### PR DESCRIPTION
**Pull request by marcelveldt**
Automatic version bumps to major, minor or patch based on the labels used within PR's.

@ludeeus @MartinHjelmare

### CODE REVIEW

**Comment by marcelveldt** [comment]
Also added the workflow for the PR labels to be mandatory

File: .github/workflows/pr-labels.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+    types:
+      - opened
+      - labeled
+      - unlabeled
+      - synchronize
+  workflow_call:
```
Comment (ludeeus) [inline .github/workflows/pr-labels.yaml:12]:
```suggestion
    types:
      - labeled
      - unlabeled
```
These are not needed here

**Review by MartinHjelmare — APPROVED** [review]
Side note: It would perhaps be good to be able to run the release-drafter workflow manually via `workflow_dispatch` event? That way we can re-label PRs and re-run release-drafter.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

**Comment by ludeeus** [comment]
That is one option, you can also just rerun the last run (its the same place as you have to trigger it anyway)

**Comment by marcelveldt** [comment]
So is this PR fine as-is ? We don't adjust the workflow for the labels (and accept the bit of overhead it brings) ?

File: .github/workflows/pr-labels.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: PR Labels
+
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - unlabeled
```
Comment (ludeeus) [inline .github/workflows/pr-labels.yaml:9]:
```suggestion
on:
  pull_request:
    types:
      - synchronize
      - labeled
      - unlabeled
    branches:
      - main
```

**Review by ludeeus — APPROVED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**